### PR TITLE
feat: Add new completion layer for tubing & screen

### PR DESCRIPTION
**Pull request by ovineq**
Ported from wellx-designer

image
image
image

### CODE REVIEW

**Comment by ovineq** [comment]
Corresponding data needs to be merged first: https://github.com/equinor/esv-intersection-data/pull/9

File: .storybook/src/data.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  poslog,
+  cementData,
+  holeSizeData,
+  casingData,
+  surfaces,
+  completion,
+  stratColumns,
+  picks,
+  seismic,
+  completionV2,
```
Comment (ooystein) [inline .storybook/src/data.ts:11]: Let delete existing completion data in esv-intersection-data repo and place our new data there. So we call it "completion" not "completionV2"

File: src/layers/CompletionLayerV2.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  radius: number;
+}
+
+export interface CompletionLayerV2Options<T extends Completion[]> extends WellComponentBaseOptions<T> {
+  screenScalingFactor?: number;
+  screenLineColor?: number;
+  tubingScalingFactor?: number;
+}
+
+export class CompletionLayerV2<T extends Completion[]> extends WellboreBaseComponentLayer<T> {
```
Comment (ooystein) [inline src/layers/CompletionLayerV2.ts:22]: Use CompletionLayer as name. Old layer was never implemented and was just acting as a placeholder.

**Review by ooystein — CHANGES_REQUESTED** [review]
Looking good, a couple small adjustments and it should be ready.

**Review by ooystein — APPROVED** [review]

**Pull request merged**